aiida/manage/configuration/config.py

```diff
@@ -28,7 +28,7 @@
 SCHEMA_FILE = 'config-v9.schema.json'
 
 
-@lru_cache(1)
```

Comment:
I take it this is not strictly related, but I guess it couldn't hurt.

Reply:
Yes, not related. `@cache` was added in Python 3.9 so we can now use it since the 3.8 support was dropped.